types/redux-actions/index.d.ts

```diff
@@ -165,19 +170,19 @@ export function handleActions<StateAndPayload>(
     reducerMap: ReducerMap<StateAndPayload, StateAndPayload>,
     initialState: StateAndPayload,
     options?: Options
-): Reducer<StateAndPayload, StateAndPayload>;
+): ReduxReducer<StateAndPayload, StateAndPayload>;
```

Comment:
Same here, inialState is not allowed to be undefined, why reducer may receive undefined state?

Reply:
Disregard this